megamek/src/megamek/common/Mech.java

```diff
@@ -972,29 +973,11 @@ public int getWalkHeat() {
         return extra + (hasEngine() ? getEngine().getWalkHeat(this) : 0);
     }
 
-    /**
-     * Returns whether this mech should use conditional ejection
-     *
-     * @return
-     */
-    /*
-     * public boolean shouldUseConditionalEject() { if (game !=null &&
-     * game.getOptions().booleanOption(OptionsConstants.RPG_CONDITIONAL_EJECTION)) { return true; }
-     *
-     * return false; }
-     */
-
-    /*
-     * (non-Javadoc)
-     *
-     * @see megamek.common.Entity#getRunMP(boolean, boolean, boolean)
-     */
     @Override
-    public int getRunMP(boolean gravity, boolean ignoreHeat,
-            boolean ignoreModularArmor) {
+    public int getRunMP(boolean gravity, boolean ignoreHeat, boolean ignoreModularArmor) {
         MPBoosters mpBoosters = getArmedMPBoosters();
-        if (mpBoosters.hasMASCAndOrSupercharger()) {
-            return mpBoosters.calcRunMP(
+        if (!mpBoosters.isNone()) {
```

Comment:
Same here. a `isAvailable()` method would help here instead of negating a `isNone()` one. The call can be the same internally but it shows intent better than negation.

Reply:
I disagree, and find redirect methods to cause more confusion than basic boolean logic

Reply:
I've found them to be helpful regardless of language. It's still Boolean logic it just reads in a way that makes it easier for newer people (and even experienced once) to read.